release-notes/3.0/preview/api-diff/preview7/3.0-preview7_System.Threading.md

```diff
@@ -0,0 +1,17 @@
+# System.Threading
+
+``` diff
+ namespace System.Threading {
+     public sealed class PreAllocatedOverlapped : IDisposable {
++        ~PreAllocatedOverlapped();
+     }
+     public sealed class ReaderWriterLock : CriticalFinalizerObject {
+-        ~ReaderWriterLock();
```

Comment:
When did ReaderWriterLock have a finalizer?

Reply:
Looks like this came from https://github.com/dotnet/corefx/pull/38190/files